tslint.json

```diff
@@ -37,8 +37,9 @@
       "check-module"
     ],
     "no-implicit-dependencies": [
-      true,
-      "dev"
+      false,
+      "dev",
+      ["@brightside/imperative"]
```

Comment:
This will be change to `true, dev, imperative` by #288 
Sorry for the confusion
No change required!

Reply:
We should double check prior to the next version to make sure we haven't overwritten the correct final implementation

Reply:
as long as this PR goes in before #288, we should be fine.
But agreed, let's double check once everything is on master :)